dhis-2/dhis-services/dhis-service-validation/pom.xml

```diff
@@ -34,6 +34,10 @@
       <groupId>org.hisp.dhis</groupId>
       <artifactId>dhis-service-schema</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Ig we remove the injection of `SystemSettingManager systemSettingManager` in the Validator, can this be removed? Or is it still needed?

Reply:
see above